tracer/src/Datadog.Trace/SpanTypes.cs

```diff
@@ -76,5 +76,10 @@ public static class SpanTypes
         /// The span type for serverless integrations.
         /// </summary>
         public const string Serverless = "serverless";
+
+        /// <summary>
+        /// The span type for couchbase integrations.
+        /// </summary>
+        public const string Couchbase = "couchbase";
```

Comment:
Span types should be a short list of general types like `web` (server), `http` (client), `sql`, `cache`, etc. These values have special meaning in the agent and the backend. Anything else is treated as a "custom" type (no special meaning). We've made mistakes using some specific product names (like `redis` and mongodb`), but we should not keep digging that hole any deeper. For Couchbase, I'm not sure if this should be `nosql` or `no-sql`. We should check our docs or ask the larger APM team.

Reply:
Switch to "db" as it is what java is doing

Reply:
@pierotibou and I talked offline. `db` looks like the best option since it classifies these spans under the "DB" filters in the UI. We should consider updating the Aerospike integration to do the same (in a separate PR/issue). `sql` and `mongodb` have special meaning for obfuscation in the agent so those are separate from `db`.